web/api/web_api_v1.c

```diff
@@ -83,6 +83,17 @@ void web_client_api_v1_init(void) {
         api_v1_data_google_formats[i].hash = simple_hash(api_v1_data_google_formats[i].name);
 
     web_client_api_v1_init_grouping();
+
+	uuid_t uuid;
+
+	// generate
+	uuid_generate(uuid);
+
+	// unparse (to string)
+	char uuid_str[37];
+	uuid_unparse_lower(uuid, uuid_str);
+
+	api_secret = config_get(CONFIG_SECTION_WEB, "api authorization token", uuid_str);
```

Comment:
I changed it, but had to include it in rrdhost, because the registry is initialized after the web server and client. I have a question regarding streaming, we'll talk about it.

Reply:
Moved it out of rrdhost. The call to initialize it is made at the end of rrd_init.